superset-frontend/src/pages/SavedQueryList/index.tsx

```diff
@@ -216,10 +216,10 @@ function SavedQueryList({
   };
 
   const copyQueryLink = useCallback(
-    (id: number) => {
+    (uuid: string) => {
```

Comment:
Does TypeScript support a UUID type?

Reply:
not that I'm aware of, didn't find anything on a quick google search either